tokens/properties/mixins/BouncingDots.json

```diff
@@ -7,12 +7,12 @@
             "value": "{border.radius.circle.value}"
         },
         "margin-left": {
-            "value": "{dimension.dynamic.size.100.value}/4",
-            "comment": "not sure the operation is going to work"
+            "value": "{dimension.dynamic.size.25.value}",
```

Comment:
I was pondering whether it makes more sense to use a hardcoded value here instead of a token just in case the ration between these two token changes, but it seems safe in this case.